app/presenters/url.rb

```diff
@@ -0,0 +1,22 @@
+class Url
+  include Rails.application.routes.url_helpers
+  include ActionView::Helpers::UrlHelper
+
+  attr_reader :link_text, :path_name, :params
+
+  def initialize(link_text:, path_name:, params: {})
+    @link_text = link_text
+    @path_name = path_name
+    @params = params
+  end
+
+  def to_s
+    link_to link_text, url
+  end
+
+  private
+
+  def url
+    send("#{path_name}_path", params)
```

Comment:
Would reek get mad if we didn't mix in the URL helpers and just called methods on them directly?

```ruby
Rails.application.routes.url_helpers.send(:root_url)
# => "http://localhost:3000/"
```